subprojects/zlib.wrap

```diff
@@ -8,3 +8,6 @@ source_hash = c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1
 patch_url = https://wrapdb.mesonbuild.com/v1/projects/zlib/1.2.11/4/get_zip
 patch_filename = zlib-1.2.11-4-wrap.zip
```

Comment:
This should probably just be bumped with wraptool rather than manually added; it looks like we're out of date.
```
ryan@kenshin:~/git/libass$ meson wrap info zlib
Available versions of zlib:
  1.2.11 5
  1.2.11 4
  1.2.11 3
  1.2.11 2
  1.2.11 1
  1.2.8 8
  1.2.8 7
  1.2.8 6
  1.2.8 4
  1.2.8 2
```